documentation/creating-components/tutorial-extending-component.asciidoc

```diff
@@ -6,33 +6,27 @@ layout: page
 
 = Extending Components
 
-Extending a component is an easy way of creating a new component.
+You can create a new component by extending any existing component.
 
-Vaadin flow applications are formed of many web components.
-In most of the cases, those components have a client-side component and a server-side component.
+For most components, there is a client-side component and a corresponding server-side component: 
 
-The client-side component contains the HTML, CSS, and JavaScript.
-This client component defines a set of properties that define the behaviour of the component.
+* *Client-side component*: Contains the HTML, CSS, and JavaScript, and defines a set of properties that determine the behavior of the component on the client side.
 
-The server-side component allows modifying those properties to manage the component's behaviour from Java (server side).
+* *Server-side component*: Contains Java code that allows for changing of the client-side properties, and manages the component behavior on the server side.
```

Comment:
```suggestion
* *Server-side component*: Contains Java code that allows for changing of the client-side properties, and manages the component behavior on the server side.

You can extend a components either using server-side or client-side approaches. In this chapter we'll present two different approaches to do the same modification for the built-in text field component.
```

I think we need clarify that this are alternative ways to accomplish the same thing. I'll propose some related changes for the subtitles too, but feel free to stylish. I think you'll get the point.